libsolidity/ast/Types.cpp

```diff
@@ -1420,8 +1420,7 @@ bool ContractType::isExplicitlyConvertibleTo(Type const& _convertTo) const
 {
 	return
 		isImplicitlyConvertibleTo(_convertTo) ||
-		_convertTo.category() == Category::Integer ||
-		_convertTo.category() == Category::Contract;
+		_convertTo == IntegerType(160, IntegerType::Modifier::Address);
```

Comment:
Perhaps we should introduce constants for the more common elementary types so that we can use
`_convertTo == ElementaryTypes::Address` here.

Reply:
Maybe right - I count 14 occurrences of this type alone...